src/material-moment-adapter/adapter/moment-date-adapter.ts

```diff
@@ -48,7 +73,8 @@ export class MomentDateAdapter extends DateAdapter<Moment> {
     narrowDaysOfWeek: string[]
   };
 
-  constructor(@Optional() @Inject(MAT_DATE_LOCALE) dateLocale: string) {
+  constructor(@Optional() @Inject(MAT_DATE_LOCALE) dateLocale: string,
+    @Inject(MAT_MOMENT_DATE_ADAPTER_OPTIONS) private options: MatMomentDateAdapterOptions) {
```

Comment:
In order to make this not a breaking change for anyone who might be extending the `MomentDateAdapter` lets make this:

```ts
/** @deletion-target 8.0.0 make this param required */
@Optional() @Inject(MAT_MOMENT_DATE_ADAPTER_OPTIONS) private options?: MatMomentDateAdapterOptions
```